kalite/coachreports/models.py

```diff
@@ -36,20 +35,6 @@ def get_user_logs(cls, user, pl_video_ids=None, pl_exercise_ids=None):
 
         return (user_vid_logs, user_ex_logs)
 
-    @classmethod
```

Comment:
Quizzes are awesome, but we should probably bring them back in another form, rather than attempting to raise the dead.